Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -586,12 +613,108 @@ class SyncedTabsButton: UIButton {
         super.updateConstraints()
 
         title.snp_remakeConstraints { make in
-            make.left.equalTo(image.snp_right).offset(TwoLineCellUX.BorderViewMargin)
+            make.leading.equalTo(image.snp_trailing).offset(TwoLineCellUX.BorderViewMargin)
             make.centerY.equalTo(self).offset(descriptionLabel.hidden ? 0 : -10)
         }
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+}
+
+class RecentlyClosedTabsButton: UIButton {
+    private let ImageMargin: CGFloat = 12
+
+    override var enabled: Bool {
+        didSet {
+            self.alpha = enabled ? 1.0 : 0.5
+        }
```

Comment:
You're overriding the original implementation, so you probably want to add a `super.enabled = enabled` here.